### DISCUSSION TITLE
How to Retrieve Paginated Records with Total Count in Supabase using C#?

### DISCUSSION THREAD

**Discussion by bhagavan44** [supabase/supabase#28824] [Questions]
Hi everyone,

I'm working on a project where I need to fetch a paginated list of records from Supabase using C#. Along with the paginated data, I also need to get the total count of records that match the query (for pagination purposes).

I've been able to retrieve data from Supabase, but I'm not sure how to structure the query or the response to include both the paginated records and the total count. Could someone provide guidance or an example of how to accomplish this?

Thanks in advance!

**Comment by bhagavan44** [comment]
We have this for js client, not sure how to use this in c#
`const { error, data, count } = await supabase
    .from('table')
    .select('*', { count: 'exact' })`

**Comment by oscarthroedsson** [comment]
@bhagavan44 Did you get an answer somewhere else? I am trying to do the same thing
